canvas_account_admin_tools/settings/base.py

```diff
@@ -263,9 +263,17 @@
 ICOMMONS_REST_API_TOKEN = SECURE_SETTINGS.get('icommons_rest_api_token')
 ICOMMONS_REST_API_SKIP_CERT_VERIFICATION = False
 
-PERMISSION_ACCOUNT_ADMIN_TOOLS = 'account_admin_tools'
-PERMISSION_PEOPLE_TOOL = 'people_tool'
-PERMISSION_XLIST_TOOL = 'cross_listing'
+# TOOL_PERMISSIONS is used to map app names to SchoolPermission.permission names
+# to determine visibility of tools on dashboard and access to tool views/pages
+TOOL_PERMISSIONS = {
+    # NOTE: canvas_account_admin_tools permission name is shortened to
+    # account_admin_tools due to SchoolPermission.permission field max length
+    'canvas_account_admin_tools': 'account_admin_tools',  # dashboard
+    'conclude_courses': 'conclude_courses',  # external tool
+    'course_info': 'course_info',
+    'cross_list_courses': 'cross_list_courses',
```

Comment:
Shouldn't this be 'cross_listing' as per https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/pull/170/files#diff-6657219d5c6186d4b0ad46cf3bd746dfL268?

[For the dynamin query to find existing cross listing permissions?]